plugins/maven/src/main/java/org/wildfly/swarm/plugin/maven/PackageMojo.java

```diff
@@ -51,9 +51,9 @@
 )
 public class PackageMojo extends AbstractSwarmMojo {
 
-    static final String UBERJAR_SUFFIX = "-thorntail";
+    static final String UBERJAR_SUFFIX = "thorntail";
 
-    static final String HOLLOWJAR_SUFFIX = "-hollow" + UBERJAR_SUFFIX;
+    static final String HOLLOWJAR_SUFFIX = "hollow" + UBERJAR_SUFFIX;
```

Comment:
So previously the hollow JAR suffix was supposed to be `hollow-thorntail`, but that's now `hollowthorntail`. Not sure if that's an issue or not.

Reply:
It's definitely an issue. `myapp-hollow-swarm.jar` should be now `myapp-hollow-thorntail.jar`.